app/models/expiry_scanner.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+require 'find'
+require 'openssl'
+require 'date'
+
+class ExpiryScanner < ApplicationRecord
```

Comment:
Does this actually need to inherit from ApplicationRecord? Seems like it could be a standalone class

Reply:
Yes, it's just a habit.